internal/apis/certmanager/validation/certificate_test.go

```diff
@@ -641,9 +658,7 @@ func TestValidateCertificate(t *testing.T) {
 							"cert-manager.io/allow-direct-injection": "true",
 						},
 					},
-					IssuerRef: cmmeta.ObjectReference{
-						Name: "invalid",
-					},
+					IssuerRef: validIssuerRef,
```

Comment:
It's a copy paste mistake by the original author I think! "invalid" is a perfectly valid name for an issuer. This is the reason I'm changing to the validIssuerRef object - because mistakes like this are easy to have creep in

> invalid with disallowed 'CertificateSecretTemplate' annotations

Is the name of the test - it's a badly named test if the issuerRef is meant to be invalid!